tests/tracee_test.go

```diff
@@ -126,3 +129,92 @@ func TestWebhookIntegration(t *testing.T) {
 	t.Log("Terminating the Tracee container...")
 	assert.NoError(t, exec.Command("docker", "kill", containerID).Run())
 }
+
+type traceeContainer struct {
+	testcontainers.Container
+}
+
+func setupTraceeContainer(ctx context.Context, tempDir string) (*traceeContainer, error) {
```

Comment:
Tracee configuration might affect the signatures. should we configure tracee similarly to how the entrypoint would?

Reply:
Yeah agreed configuration should be a test parameter too. What other params are should we add here?

Reply:
you are running the container through the entrypoint so that's good